### ISSUE TITLE
Please add language tags for labels and definitions

### ISSUE THREAD

**Issue by neilotte** [CommonCoreOntology/CommonCoreOntologies#61]
I would like to recommend that english language tags be added to every rdfs:label in cco. 

If you concur, you may find that the following query works:

```
DELETE {?s rdfs:label ?label}
INSERT {?s rdfs:label ?LabelWithLanguageTag }
WHERE {
?s ?p ?o.
?s rdfs:label ?label. 
BIND(STRLANG( ?label , "en") AS ?LabelWithLanguageTag ) .
}
```

**Comment by rorudn** [comment]
It makes sense to do this when labels and definitions are added in languages besides English. I cannot think of one but perhaps you have some use case in mind that would benefit from performing this exercise now? If so, please provide.

**Comment by neilotte** [comment]
No particular cases come to mind. But I suspect many extensions of the common core will involve a need to accommodate robust language support. For such uses, if the CCO doesn't already have these English language tags in place, then every extension would require adding them. 

As a value add, I think publishing CCO with language tags will be useful for these purposes, and I don't see how it would hurt to have them in cases where users don't need them.

**Comment by aellenhicks** [comment]
There are times when we will want to query our entire ontology suite for classes with an English label. The most straightforward way to write this is to specify the value of a language tag. If there is no value in the language tag, then the IRI will not be retrieved even if the label is in English. Not having language tags in CCO would either result in false negatives for our queries (i.e., all of the CCO resources would be missing) or else the need to write ad hoc, and therefore not generalizable. This places an extra requirement on our soft-ware developers to have intimate knowledge about which of our ontologies use language tags and which do not. Adding the language tags would help reduce this extra cognitive overhead for them and thereby help integrate CCO more robustly into our information systems. This would likely apply to other folks who are using CCO in teams with non-ontology experts and with multi-lingual data.

**Comment by rorudn** [comment]
Understood. We'll do some testing to make certain that there's no chance to break queries or applications that aren't searching for tagged values and if none are found we'll add the tags.

**Comment by mark-jensen** [comment]
Fixed with d4ea051
